service/src/main/java/datawave/microservice/authorization/AuthorizationOperationsV1.java

```diff
@@ -46,27 +46,32 @@ public AuthorizationOperationsV1(JWTTokenHandler tokenHandler, CachedDatawaveUse
         this.busProperties = busProperties;
     }
     
-    @ApiOperation(value = "Authorizes the calling user to produce a JWT value",
-                    notes = "The returned JWT can be passed to other calls in a header. For example: \"Authorization: bearer <JWT value>\".\n"
-                                    + "The user can be determined with from the supplied client certificate or trusted headers ("
-                                    + "X-SSL-clientcert-subject/X-SSL-clientcert-issuer).")
+    // convert DatawaveUser to DatawaveUserV1 and remove final caller if there are any proxied users
```

Comment:
What does DatawaveUserV1 mean?  Is it possible to name this method better: e.g. putCurrentUserIntoProxiedChain or something like that?  Lets try to make this obvious since we are not choosing to create a different object.

Reply:
How about we call this getProxiedUsers and have it return a List<DatawaveUser> instead.  That might make this a little more obvious.

Reply:
Discussed offline....